omnibus/config/software/datadog-agent-mac-app.rb

```diff
@@ -12,8 +12,14 @@
 build do
     app_temp_dir = "#{install_dir}/Datadog Agent.app/Contents"
     mkdir "#{app_temp_dir}/Resources"
+    mkdir "#{app_temp_dir}/Frameworks"
+
     copy "Agent.icns", "#{app_temp_dir}/Resources/"
 
+    # Add swift runtime libs in Frameworks (same thing a full xcode build would do for a GUI app).
+    # They are needed for the gui to run on MacOS 10.14.3 and lower
+    command "$(xcrun --find swift-stdlib-tool) --copy --scan-executable \"#{app_temp_dir}/MacOS/gui\" --scan-folder \"#{app_temp_dir}/Frameworks\" --platform macosx --destination \"#{app_temp_dir}/Frameworks\" --strip-bitcode --strip-bitcode-tool $(xcrun --find bitcode_strip)"
```

Comment:
Out of curiosity, is this having an impact on the artifact size? Don't bother if that will take time to check this out.

Reply:
Before, the `gui` executable was 9.8MB (since it statically linked all the libs we now embed). Now, the `gui` app is 117KB, and the `Frameworks` folder containing the swift libs weighs ~11MB, so there about a 1 to 2MB size increase for the package.